source/plugins/hestia/dimmableLight/DimmableLight.py

```diff
@@ -17,3 +17,7 @@ def name(self):
     def pluginType(self):
         return "Light"
 
+    @staticmethod
+    def getEmptyRequiredInfo():
+        return {"ip": "127.0.0.1", "port": "0"}
+
```

Comment:
I think getDefaultRequiredInfo would indeed be better I will change that and then merge it